### PR TITLE
Deploy in WeChat official account

### PR DESCRIPTION
**Pull request by abhishek-iiit**
## Added deploy in WeChat official account
Screenshot from 2021-07-24 11-09-03
Screenshot from 2021-07-24 11-09-20
Screenshot from 2021-07-24 11-09-46
Screenshot from 2021-07-24 11-09-53

### CODE REVIEW

**Comment by abhishek-iiit** [comment]
Linked to issue #896

File: docusaurus/docs/howto/wechat-offical.mdx [hunk 1]
```diff
@@ -0,0 +1,151 @@
+---
+title: Deploy in WeChat Official Account
+---
+
+WeChat Official Account is a perfect enforcement of the concept of SoLoMo(Social, Local, Mobile) Marketing and has tonnes of features to interact with the audience. In this how to guide, you will learn how to deploy the bot in WeChat Official Account locally and integrate bot from the list of example present in [starter templete](https://github.com/wechaty/wechaty-getting-started).
```
Comment (Rohitesh-Kumar-Jain) [inline docusaurus/docs/howto/wechat-offical.mdx]: examples

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

**Review by proudofsimin — APPROVED** [review]
This is great. Thank you.

**Review by vasvi-sood — APPROVED** [review]
This PR is awesome: )

**Review by huan — APPROVED** [review]
LGTM.

Please read and follow #1257, and feel free to help yourself to merge it by following the merge workflow.

Thank you very much!

**Comment by Rohitesh-Kumar-Jain** [comment]
@vasvi-sood and @abhishek-iiit pls solve the merging conflict, and help yourself in merging the PR

**Comment by abhishek-iiit** [comment]
Merging PR with no conflicts and 4 approvals :)

**Pull request merged**